test/lib/commands/ping.js

```diff
@@ -23,9 +26,9 @@ t.test('with details', async t => {
   registry.ping({ body: { test: true } })
   await npm.exec('ping', [])
   t.match(logs.notice, [
-    ['PING', 'https://registry.npmjs.org/'],
-    ['PONG', /[0-9]+ms/],
-    ['PONG', '{\n  "test": true\n}'],
+    `PING https://registry.npmjs.org/`,
+    /PONG [0-9]+ms/,
+    `PONG {\nPONG   "a": 1,\nPONG   "b": 2\nPONG }`,
```

Comment:
Why did this change?

Reply:
I think this is another case of old tests not reflecting reality. These logs are prefixed with `PONG` on every line.

```sh
❯ npm ping
npm notice PING https://registry.npmjs.org/
npm notice PONG 177ms
npm notice PONG {
npm notice PONG   "someDataReturnedFromPing": true
npm notice PONG }
❯ npm ping --json
npm notice PING https://registry.npmjs.org/
npm notice PONG 171ms
{
  "registry": "https://registry.npmjs.org/",
  "time": 171,
  "details": {
    "someDataReturnedFromPing": true
  }
}
```

Reply:
Why are we logging `"a": 1` instead of the actual ping response body?

Reply:
```
$ node . ping --json
npm notice PING https://registry.npmjs.org/
npm notice PONG 173ms
{
  "registry": "https://registry.npmjs.org/",
  "time": 173,
  "details": {
    "a": 1,
    "b": 2
  }
}
```

```
~/D/n/c/b/lk_remove-npmlog (lk/remove-npmlog|✚1) $ npm ping --json
npm notice PING https://registry.npmjs.org/
npm notice PONG 188ms
{
  "registry": "https://registry.npmjs.org/",
  "time": 188,
  "details": {}
}
```

Reply:
```sh
$ node . ping
npm notice PING https://registry.npmjs.org/
npm notice PONG 182ms
npm notice PONG {
npm notice PONG   "a": 1,
npm notice PONG   "b": 2
npm notice PONG }
$ npm ping
npm notice PING https://registry.npmjs.org/
npm notice PONG 175ms
```